speech/cloud-client/beta_snippets.py

```diff
@@ -46,7 +46,6 @@ def transcribe_file_with_enhanced_model(speech_file):
     audio = speech.types.RecognitionAudio(content=content)
     config = speech.types.RecognitionConfig(
         encoding=speech.enums.RecognitionConfig.AudioEncoding.LINEAR16,
-        sample_rate_hertz=8000,
```

Comment:
Any particular reason for omitting this? For WAV files, the API can infer this, but in the general case it's probably a good idea to include the sample rate

Reply:
I keep going back and forth about it. Sometimes it is useful, and other times it is causing an error when the input file has a different sample rate.